src/SerializeSourceAnnotations.cc

```diff
@@ -0,0 +1,49 @@
+#include "SerializeSourceAnnotations.h"
+#include "SourceAnnotation.h"
+#include "SerializeSourcemap.h"
+
+#include <stdio.h>
+
+using namespace drafter;
+
+static sos::Object WrapLocation(const mdp::BytesRange& range)
+{
+    sos::Object location;
+
+    location.set(SerializeKey::AnnotationLocationIndex, sos::Number(range.location));
+    location.set(SerializeKey::AnnotationLocationLength, sos::Number(range.length));
+
+    return location;
+}
+
+static sos::Object WrapAnnotation(const snowcrash::SourceAnnotation& annotation)
+{
+    sos::Object object;
+
+    object.set(SerializeKey::AnnotationCode,     sos::Number(annotation.code));
+    object.set(SerializeKey::AnnotationMessage,  sos::String(annotation.message));
+    object.set(SerializeKey::AnnotationLocation, WrapCollection<mdp::BytesRange>()(annotation.location, WrapLocation));
+
+    return object;
+}
+
+sos::Object drafter::WrapSourceAnnotations(const snowcrash::Report& report, const snowcrash::SourceMap<snowcrash::Blueprint>& sourceMap)
+{
+    sos::Object object;
+
+    object.set(SerializeKey::AnnotationsVersion, sos::String(AST_ANNOTATION_SERIALIZATION_VERSION));
+    
+    sos::Object ast;
+    ast.set(SerializeKey::ASTVersion, sos::String(AST_SERIALIZATION_VERSION));
+    object.set(SerializeKey::Ast, ast);
```

Comment:
Oh I miss `...` in https://github.com/apiaryio/api-blueprint-ast/blob/zdne/elements-ast/Parse%20Result.md

AST member, probably, should contain fully parsed AST. 
@zdne  isn't it?